index.html

```diff
@@ -325,5 +325,15 @@ <h6 class="languages-world">Languages</h6>
         </form>
       </div>
     </footer>
+
+    <div class="menu-bar">
+         <i class="fa fa-times"></i>
+        <ul class="header-slide-links">
+          <li><a href="#">Portifolio</a></li>
+          <li><a href="#">About</a></li>
+          <li><a href="#">Contact</a></li>
+        </ul>
```

Comment:
- [ ] Kindly make when the user press any of the mobile menu options, a correct part of the page is displayed. then, the menu closed.